packages/server-wallet/e2e-test/pong/controller.ts

```diff
@@ -17,29 +46,28 @@ export default class PongController {
     // types of messages between parties. e2e-test uses wire-format
     const convertedSignedStates = (signedStates as unknown) as SignedState[];
 
-    const {channelResults, outbox} = await this.wallet.pushMessage({
+    const {
+      channelResults: [pingsUpdate],
+    } = await this.wallet.pushMessage({
       signedStates: convertedSignedStates,
       to,
       from,
     });
 
-    if (!channelResults) throw Error('sanity check');
+    // FIXME: At this point, pingsUpdate is undefined, but should be a result
+    // waiting on https://github.com/statechannels/statechannels/pull/2370
 
-    for (const notification of outbox) {
-      if (notification.method === 'ChannelProposed') {
-        console.log('Observed a ChannelProposed event');
-      }
-    }
+    if (!pingsUpdate)
+      return {
+        recipient: from,
+        sender: this.myParticipantID,
+        data: {signedStates: [], objectives: []},
+      };
```

Comment:
Is this meant to be placeholder code for something?

Reply:
Yes, for `joinChannel`